README.md

```diff
@@ -1,3 +1,25 @@
 # Simple Casper
 
 Layer-2 Casper daemon for POC4.
+
+## Setup
+### Requirements
+- `Python2.7`
+- `automake`
+- `libffi`
+- `tmux`
+
+### Getting started
+Make sure you have installed the requirements, and have created a virtual enviornment.
+Now install the project dependencies:
+```
+$ pip install -r requirements.txt
+$ python setup.py install
+```
+
+Next run the Casper simulator!
+```
+$ source ./tmux.sh
```

Comment:
if you use `ZSH`, but `./tmux.sh` has `#!` of `BASH`,
then `source ./tmux.sh` is invoked with `zsh`,
but invoke `./tmux.sh` directly will invoke with `BASH`

I did not read the context, just see this snippet of words and then comment.